src/couch/src/couch_changes.erl

```diff
@@ -556,9 +552,15 @@ send_changes(Acc, Dir, FirstRound) ->
     end.
 
 
-can_optimize(true, {doc_ids, _Style, DocIds})
-        when length(DocIds) =< ?MAX_DOC_IDS ->
-    {true, fun send_changes_doc_ids/6};
+can_optimize(true, {doc_ids, _Style, DocIds}) ->
+    MaxDocIds = list_to_integer(
+        config:get("changes", "max_doc_ids", "100")
```

Comment:
I'd rather see this under `[couchdb]` than `[changes]` especially since it's a change to the couch app.